C-Plus-Plus/other/Phone_keypad.cpp

```diff
@@ -0,0 +1,64 @@
+
+#include <iostream>
+#include <string>
```

Comment:
Add one single header file, i.e **bits/stdc++.h** and remove the these two headers